docker-build.sh

```diff
@@ -96,9 +96,13 @@ _get_stages_image_name() {
   echo "${INPUT_STAGES_IMAGE_NAME:-${INPUT_IMAGE_NAME}-stages}"
 }
 
+_get_stages_image_tag() {
+  echo "${INPUT_STAGES_IMAGE_TAG:-latest}"
```

Comment:
this would make all stages to have tags like latest-1, latest-2, ... under legacy behavior (no buildkit) which I don't see it better than current behav.